.pylintrc

```
@@ -21,7 +21,10 @@ persistent=yes
 # List of plugins (as comma separated values of python modules names) to load,
 # usually to register additional checkers.
 load-plugins=pylint.extensions.docparams, # enable checking of docstring args
-    pylint.extensions.docstyle  # basic docstring stle checks
+    pylint.extensions.docstyle,  # basic docstring stle checks
```

If we're touching this line anyway, could we fix the typo? 😊 
```suggestion
    pylint.extensions.docstyle,  # basic docstring style checks
```

---

I have no permission to touch this branch, so I fix it here f4a64d47. Thanks!